heat_infoblox/object_manipulator.py

```diff
@@ -70,13 +106,30 @@ def pre_provision_member(self, member_name,
                              licenses=None):
         if licenses is None:
             licenses = []
+        if not isinstance(licenses, list):
+            licenses = [licenses]
         extra_data = {'pre_provisioning': {
             'hardware_info': [{'hwmodel': hwmodel, 'hwtype': hwtype}],
             'licenses': licenses}
         }
         self._update_infoblox_object('member', {'host_name': member_name},
                                      extra_data)
 
+    def pre_provision_ha_pair(self, member_name,
```

Comment:
It is better to add new flag like 'is_ha' to 'pre_provision_member' method then to create new method which is almost the same as 'pre_provision_member'